docs/bpftool-gen.rst

```diff
@@ -208,6 +208,20 @@ OPTIONS
 		  not use the majority of the libbpf infrastructure, and does not need
 		  libelf.
 
+	-B, --base-btf *FILE*
+		  Pass a base BTF object. Base BTF objects are typically used
+		  with BTF objects for kernel modules. To avoid duplicating
+		  all kernel symbols required by modules, BTF objects for
+		  modules are "split", they are built incrementally on top of
+		  the kernel (vmlinux) BTF object. So the base BTF reference
+		  should usually point to the kernel BTF.
+
+		  When the main BTF object to process (for example, the
+		  module BTF to dump) is passed as a *FILE*, bpftool attempts
+		  to autodetect the path for the base object, and passing
+		  this option is optional. When the main BTF object is passed
+		  through other handles, this option becomes necessary.
```

Comment:
This paragraph would need to be adjusted, it's written with `bpftool btf dump` in mind. From what I understand, if we don't pass `-B` for `bpftool gen min_core_btf`, we just don't use split BTF, right? There's no attempt to guess the path. So this paragraph can maybe just be deleted.